scripts/module/gen_detectors.sh

```diff
@@ -16,4 +16,3 @@ for detector in ${TARGET}/conf/[0-9][0-9]-*.yaml; do
         j2 ./scripts/templates/$(echo ${dst} | sed 's/detectors/detector/').tf.j2 ${detector} >> ${TARGET}/${dst}-gen.tf
     done
 done
-
```

Comment:
why?

Reply:
Editor configuration issue ... Should be good now.